src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Models/Course.cs

```diff
@@ -29,7 +30,7 @@ public class Course
         public IEnumerable<CourseRun> CourseRuns { get; set; }
         public IEnumerable<string> ValidationErrors { get; set; }
         public IEnumerable<BulkUploadError> BulkUploadErrors { get; set; }
-        public int CourseStatus { get; set; }
+        public CourseStatus CourseStatus { get; set; }
```

Comment:
Are these changes, from what appears to be `ints` to `enums`, just housekeeping?

Reply:
Pretty much; it's bringing it into line with the other Cosmos types.